protonvpn_cli/constants.py

```diff
@@ -6,11 +6,44 @@
 except KeyError:
     USER = getpass.getuser()
 
-CONFIG_DIR = os.path.join(os.path.expanduser("~{0}".format(USER)), ".pvpn-cli")
+# Ensure backwards compatibility
+home_path = os.path.expanduser("~{0}".format(USER))
+home_config = os.path.join(home_path, ".pvpn-cli")
+if os.path.exists(home_config):
+    CONFIG_DIR = home_config
+    DATA_DIR = home_config
+    CACHE_DIR = home_config
+
+else:
+    # Get the config directory
+    xdg_config_str = os.getenv("XDG_CONFIG_HOME")
+    if not xdg_config_str:
+        xdg_config = os.path.join(home_path, ".config")
+    else:
+        xdg_config = os.path.realpath(xdg_config_str)
```

Comment:
nit: perhaps you can simplify it by writing

```python
xdg_config = os.path.realpath(
    os.getenv("XDG_CONFIG_HOME", os.path.join(home_path, ".config"))
)
```

Reply:
You could also use the [XDG module](https://pypi.org/project/xdg/) (last updated April of 2021) to facilitate the base directory specification.